recipe/yum_requirements.txt

```
@@ -1,5 +1,2 @@
+#tk_needs_libx11_but_avoids_a_runtime_requirement_of_libx11_to_reduce_the_dependencies_of_python
 libx11
```

You need at least
```suggestion
libX11-devel
```
however, this kind of setup failed previously (see [logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1028624&view=logs&jobId=2aeac593-170e-54b6-7054-3a1104dd2f98) for https://github.com/conda-forge/python-feedstock/pull/706/commits/165b52f39c47aaa7a436a86e026ac841ccf2668c)